core/tests/webdriverio_utils/ExplorationEditorMainTab.js

```diff
@@ -74,7 +74,8 @@ var ExplorationEditorMainTab = function() {
   var nodeLabelLocator = '.e2e-test-node-label';
   var openOutcomeDestEditor = $('.e2e-test-open-outcome-dest-editor');
   var openOutcomeFeedBackEditor = $('.e2e-test-open-outcome-feedback-editor');
-  var postTutorialPopover = $('.ng-joyride .popover-content');
+  // eslint-disable-next-line oppia/e2e-practices
```

Comment:
Why is this added?

Reply:
![Screenshot from 2022-10-27 09-44-31](https://user-images.githubusercontent.com/77006897/198190120-286134a5-4d8d-494e-92c8-ad790ded70bd.png)

Reply:
removed